### ISSUE TITLE
dot pager ?

### ISSUE THREAD

**Issue by kbytin** [meandmax/lory#323]
I saw this ussue is closed now https://github.com/meandmax/lory/issues/13

Is it possible to have dot's pager?

**Comment by JeffWeim** [comment]
I'm wondering the same. The issue was closed before anyone showed a working example...

**Comment by Sokolua** [comment]
+100 for dots

**Comment by jeebseven** [comment]
apparently working on one

**Comment by yoanmalie** [comment]
I would use Lory instead Slick.js. But the dot pager is the only thing is missing here to replace Slick in my workflow.
+1 For this Feature request.

**Comment by jeebseven** [comment]
I'm gonna make a pullrequest within the week :)

**Comment by nstanard** [comment]
You can create your own dot pager with the provided events people.... No need to bloat the project with stuff you can add on your own.

**Comment by jeebseven** [comment]
Absolutely, but maybe not everybody knows how, but want to use the slider anyways.

TL;DR
In my opinion a great slider is one where you can 1. have a well documented api - which is here better then nothing but far from perfect - 2. you can extend it by your own code (check), and last but not least: have quite a few standardized options to pass through - like those mouse-click events. If it's about not "bloating" the project, you, I or anybody else can write this bubble navigation, autostart function, whatever as a plugin - if that is more likeminded.

**Comment by nstanard** [comment]
"have quite a few standardized options to pass through" 

Where does it end? Honestly... the thing I love about this slider is how minimal it is. I'm not a main contributor to this project so I don't have much weight to throw around but still.... programming isn't about having standardized options provided all the time. It's about understanding how things work and being able to adjust them the way you want. I think there is a gitter chat for this project and if anybody has any questions they can refer to that and ask questions.

**Comment by jeebseven** [comment]
> The more features you add the more problems you get.

I'm with you - totally - as a programmer. But as a user you can say - well may this slider is not for me if I need a dot pagination and cannot program it myself. Or whatever. Or you can at least be modular and say - well if you need a dot pagination then enable it - or just get it from that place and enable this js after lory.js has loaded. For sure you have to maintain more but as stated here it is an enhancement. The thing I love about lory is its flexibility to be whatever you want it to be. But by adding new features you do not have to break those flexibility. 

About the documentation. Well yes the Readme is rather clear which options you have and what you can do with the api and a bit of " well that does that thing" 

But for me there shall have been a bit more, whilst I was developing the dot navigation and also an autostart option - where every designer I know is after it - I have had to take a deeper look into the code to understand what really was happening - and yes the in-code documentation is very well written, even if the codestyle with those asterisks is quite new to me ^^

**Comment by nstanard** [comment]
If you open a PR and they accept it I cannot stand in the way. However, I think lory.js is fine without it and adding it simply because designers don't want to take the time to learn how to code is not a good reason.

**Comment by yoanmalie** [comment]
Hey,
@nstanard, i don't know how to do this. I don't code JS.
BTW, i agree with @jeebseven. Feel free to put your PR and let's see if the plugin creator would accept this.

**Comment by nstanard** [comment]
@yoanmalie, @jeebseven: http://kenwheeler.github.io/slick/

Have fun.

**Comment by jeebseven** [comment]
@nstanard well jQuery dependancies. Not for me :)

**Comment by yoanmalie** [comment]
@nstanard  Slick **Require JQuery**, Slick is oversized for most of my projects. Slick is 5× heavier.
Please, be relevant.

**Comment by nstanard** [comment]
http://codepen.io/nstanard/pen/jrwymd

Have fun

**Comment by nstanard** [comment]
@yoanmalie Why are you charged with maintaining sites if you don't use JS? You should have enough knowledge to setup API/Event calls to add and remove a class.

**Comment by yoanmalie** [comment]
Wow. That escalated quickly. ᕕ( ᐛ )ᕗ

@nstanard You should have enough knowledge to ensure that your example works correctly and also, avoid bugs. Before "arguing" that i should to know JS.

i repeat: please, be relevant.
The fact that i can't code JS doesn't mean i can't use a libs or whatever.
Libs are made to save times and allow lot peoples to use them easily.

As @jeebseven said,

> A great slider is one where you can:
> 1. Have a well documented api ;
> 2. You can extend it by your own code ;
> 3. **Have quite a few standardized options to pass through**.

The third point is why i'm here. IMO dot pager are sandardized for a slider like _Prev_ and _Next_ buttons.

**Comment by nstanard** [comment]
What bugs? Please be specific.

The example I provided didn't have click events on the list items and so clicking them didn't do anything. If you want that too then its' a few lines of code more.

First example with only visual feedback: http://codepen.io/nstanard/pen/jrwymd
Second example with click events on the list items using the public API method to slide to the index: http://codepen.io/nstanard/pen/EgvVpV

Any issues you see you can mention and I'll be glad to fix but that's a working example in about 38 lines of code. I'm not arguing about anything @yoanmalie. This project is fantastic and doesn't need this feature. I have provided two examples now on how to add this functionality yourself using the provided public API and custom events.
